doc/ansible/dev/testing_guide.md

```diff
@@ -3,12 +3,13 @@
 ## Getting started
 
 ### Requirements
-To begin, you sould have:
+To begin, you should have:
 - The latest version of the [operator-sdk](https://github.com/operator-framework/operator-sdk) installed.
-- Docker installed and running
+- [Docker](https://www.docker.com/get-started) installed and running
 - [Molecule](https://github.com/ansible/molecule) >= v2.20
 - [Ansible](https://github.com/ansible/ansible) >= v2.7
-- [jmespath](https://pypi.org/project/jmespath/)
+- [JMESPath](https://github.com/jmespath/jmespath.py)
```

Comment:
I think this is only required up to 0.15.x, 0.16 no longer scaffolds code using jmespath and instead uses the `wait_condition` option in the `k8s` module